pkg/epinio/detail/applications.vue

```diff
@@ -51,70 +49,141 @@ export default Vue.extend<Data, any, any, any>({
+        <!-- <span>Uptime 1d 23min</span>
+        Age: 2d ago -->
+      </template>
+
+      <!-- Resources count slot -->
+      <template v-slot:resourcesCount>
+        <!-- // TODO: Depends on https://github.com/epinio/epinio/issues/1471 -->
+        <!-- <div>
+          {{ value.configCount }} {{ t('epinio.applications.detail.counts.config') }}
+        </div> -->
+        <div>
+          {{ value.configCount }} {{ t('epinio.applications.detail.counts.services') }}
         </div>
-      </SimpleBox>
-      <SimpleBox class="envs">
-        <div class="box">
-          <h1>{{ value.envCount }}</h1>
-          <h3>
-            {{ t('epinio.applications.detail.counts.envVars') }}
-          </h3>
+        <div>
+          {{ value.envCount }} {{ t('epinio.applications.detail.counts.envVars') }}
         </div>
-      </SimpleBox>
-    </div>
-    <h3 v-if="value.deployment" class="mt-20">
+      </template>
+    </ApplicationCard>
+
+    <h3 v-if="value.deployment" class="mt-20 mb-20">
       {{ t('epinio.applications.detail.deployment.label') }}
     </h3>
 
-    <Tabbed v-if="value.deployment" class="deployment" default-tab="summary">
-      <Tab label-key="epinio.applications.detail.deployment.summary" name="summary" :weight="1">
-        <div class="simple-box-row app-instances">
-          <SimpleBox>
-            <ConsumptionGauge
-              :resource-name="t('epinio.applications.detail.deployment.instances')"
-              :capacity="value.desiredInstances"
-              :used="value.readyInstances"
-              :used-as-resource-name="true"
-              :color-stops="{ 70: '--success', 30: '--warning', 0: '--error' }"
-            >
-            </ConsumptionGauge>
-            <div class="scale-instances">
-              <PlusMinus class="mt-15 mb-10" :value="value.desiredInstances" :disabled="saving" @minus="updateInstances(value.desiredInstances-1)" @plus="updateInstances(value.desiredInstances+1)" />
-            </div>
-            <table class="stats">
+    <div class="deployment">
+      <div class="simple-box-row app-instances">
+        <SimpleBox>
+          <ConsumptionGauge
+            :resource-name="t('epinio.applications.detail.deployment.instances')"
+            :capacity="value.desiredInstances"
+            :used="value.readyInstances"
+            :used-as-resource-name="true"
+            :color-stops="{ 70: '--success', 30: '--warning', 0: '--error' }"
+          >
+          </ConsumptionGauge>
+          <div class="scale-instances">
+            <PlusMinus class="mt-15 mb-10" :value="value.desiredInstances" :disabled="saving" @minus="updateInstances(value.desiredInstances - 1)" @plus="updateInstances(value.desiredInstances + 1)" />
+          </div>
+        </SimpleBox>
+        <!-- Source information -->
+        <SimpleBox v-if="value.sourceInfo">
+          <div class="deployment__origin__row">
+            <h4>Deployment Details</h4>
```

Comment:
The switch to ul/li leaves the spacing unable to consume the correct amounts

![image](https://user-images.githubusercontent.com/18697775/170682619-ce5a51aa-7d00-4a38-9582-a5d996a43691.png)